Amethyst/default.amethyst

```diff
@@ -45,6 +45,14 @@
         "mod": "mod1",
         "key": "f"
     },
+    "focus-screen-ccw": {
+        "mod": "mod1",
+        "key": "p"
```

Comment:
I'm wary of decisions on default commands. What's the reasoning here?

Reply:
1. unused
2. close to home row, i.e. less hand movement (e.g. compared to left/right arrow keys)
3. one hand for modifier, second hand for key
4. mnemonics: next/previous. I also remapped `throw-space-right/left` to `mod2+n/p`. One could use as default `left/right` to be consistent.

Anyhow, right now I use Yabai but keep my eye on Amethyst because I am not the biggest fan of bsp (maybe my displays are not big enough). Therefore I do not work on this PR as long as I stay with Yabai. My experience with Yabai or Amethyst is that macOS is not the best OS for tiling window manager.

Reply:
That's a pretty reasonable explanation. I'll take over the PR to clean it up and get it merged. Thanks for the work so far.

> My experience with Yabai or Amethyst is that macOS is not the best OS for tiling window manager.

A painful truth.

Reply:
> I'll take over the PR to clean it up and get it merged. Thanks for the work so far.

That's great. Thank you.

> A painful truth.

Linux becomes more and more appealing. Unfortunately, I have bought a non-cross platform license of Mathematica for macOS. Also other things keep me from moving away (e.g. long term 1Password user without the new subscription based model).

Anyhow, I really appreciate projects such as Amethyst. Keep up the excellent work!